src/components/PersonDetails.js

```diff
@@ -1,6 +1,10 @@
-// Component PersonDetails should render an unordered list 
-  // unordered list should have two <li> item with below details:
-    // 1. Person's profession 
-    // 2. Person's accomplishment
-
-export default function PersonDetails({ profession, accomplishment }) {}
\ No newline at end of file
+export default function PersonDetails({ profession, accomplishment }) {
+  return(
+    <div>
+      <ul>
+        <li>{profession}</li>
+        <li>{accomplishment}</li>
```

Comment:
List elements should have been render explicit label text as Profession: and Accomplisment: (She the screenshot)
```
<li>`Profession: ${profession}`</li>
<li>`Accomplishment: ${accomplishment}`</li>
```